web/modules/custom/joinup_news/src/Controller/NewsController.php

```diff
@@ -64,14 +64,25 @@ public function add(RdfInterface $rdf_entity) {
   /**
    * Handles access to the news add form through rdf entity pages.
    *
+   * Access is granted to moderators and group members that have the permission
+   * to create news articles inside of their group, which in practice means this
+   * is granted to collection and solution facilitators.
```

Comment:
For the collection, under certain circumstances, members can also create news. It depends on the fields 'Closed' and 'eLibrary creation'.